### PR TITLE
Modified LogIn page, Home Page & created Reset Password page

### PR DESCRIPTION
**Pull request by Ade0211**
1) Login Page done with a teacher login button which links to the login page when clicked
2) Homepage done with Login inputs which redirects to the next page when the username is validated

### CODE REVIEW

**Review by Wolfjawan — APPROVED** [review]
LGTM 👍

File: src/components/LoginComponent/LoginComponent.jsx [hunk 1]
```diff
@@ -1,5 +1,85 @@
...
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(information),
+    };
+    fetch("https://stingy-cherry-sight.glitch.me/login", requestOptions)
+      .then((res) => res.json())
+      .then((res) => {
+        console.log(res);
```
Comment (Denis-Andrei) [inline src/components/LoginComponent/LoginComponent.jsx]: I think the console.log can be removed

File: src/components/LoginComponent/LoginComponent.jsx [hunk 2]
```diff
@@ -1,5 +1,85 @@
...
+        if (res.success === true) {
+          localStorage.setItem("token", res.token); //storing the token in localStorage.
+          history.push("/student-tracker");
```
Comment (Denis-Andrei) [inline src/components/LoginComponent/LoginComponent.jsx]: This route should go to /choose-class not /student-tracker

File: src/components/LoginComponent/LoginComponent.jsx [hunk 3]
```diff
@@ -1,5 +1,85 @@
...
+        <div className="buttons">
+          <input type="submit" className="login-box-btn" value="Log In" />
+        </div>
+        <ul>
+          <li>
+            <a href="#">Forgot your password ?</a>
+          </li>
+        </ul>
+      </form>
+      <footer className="page-footer">
```
Comment (Denis-Andrei) [inline src/components/LoginComponent/LoginComponent.jsx]: Footer can be removed

File: src/components/LoginComponent/logincomponent.scss [hunk 4]
```diff
@@ -0,0 +1,70 @@
+@import "../../assets/sass/main.scss";
+
+.login-box {
```
Comment (Denis-Andrei) [inline src/components/LoginComponent/logincomponent.scss:3]: Maybe will be better if you center this component in the Login Page and not here

File: src/pages/Front/front.scss [hunk 5]
```diff
@@ -1,4 +1,39 @@
...
+  &__logo {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 68rem;
+    position: relative;
+    overflow: hidden;
+  }
+  &-button {
```
Comment (Denis-Andrei) [inline src/pages/Front/front.scss]: You can create a Button component and use it in both pages( front and login)

**Review by Denis-Andrei — CHANGES_REQUESTED** [review]

File: src/components/LoginComponent/LoginComponent.jsx (hunk 1 above)
Comment (Ade0211) [reply src/components/LoginComponent/LoginComponent.jsx]: I have removed it

File: src/components/LoginComponent/LoginComponent.jsx (hunk 2 above)
Comment (Ade0211) [reply src/components/LoginComponent/LoginComponent.jsx]: I only used it for a test but i have now changed it

File: src/components/LoginComponent/logincomponent.scss (hunk 4 above)
Comment (Ade0211) [reply src/components/LoginComponent/logincomponent.scss:3]: Thanks. done

File: src/pages/Front/front.scss (hunk 5 above)
Comment (Ade0211) [reply src/pages/Front/front.scss]: Thanks for that. I have created it and made new commits

File: src/components/LoginComponent/LoginComponent.jsx (hunk 1 above)
Comment (ISTANBULBEKLE) [reply src/components/LoginComponent/LoginComponent.jsx]: Great Job Guys, you are HEROs

**Comment by ISTANBULBEKLE** [comment]
Guys, also thanks to putting many notes for us to follow

**Review by ISTANBULBEKLE — APPROVED** [review]
Great Job Guys. Go For Mars

File: src/components/LoginComponent/LoginComponent.jsx (hunk 3 above)
Comment (Ade0211) [reply src/components/LoginComponent/LoginComponent.jsx]: The footer has been removed

**Review by Denis-Andrei — APPROVED** [review]
Looks good.

**Pull request merged**